contribs/gmf/examples/layertree.html

```diff
@@ -133,10 +133,31 @@
       .treenode a.expand-node.fa[aria-expanded="true"]::before {
           content: "\f078";
       }
+
+      /** Disclaimer */
+      gmf-disclaimer {
+        position: absolute;
+        bottom: 0.5rem;
+        left: 0.5rem;
+        width: 30rem;
+      }
+      gmf-disclaimer .alert {
+        padding: 0.5rem 1rem;
+        margin: 0 0 0.5rem 0;
+      }
+      gmf-disclaimer .alert-dismissable .close,
+      gmf-disclaimer .alert-dismissible .close {
+        right: -5px;
```

Comment:
The CSS of bootstrap uses px.  What's wrong with doing the same ?  Please, let me know if you want to to absolutely use `rem` for this.